app/common/prescription/controllers/patientSimplifiedPrescriptionController.js

```diff
@@ -126,17 +126,17 @@
         vm.prescriptionItem.isArv = false;
         vm.prescriptionItem.interruptedReason = {};
         vm.prescriptionItem.isPlanInterrupted = false;
-        vm.prescriptionItem.therapeuticLine = {};
+        vm.prescriptionItem.arvPlan = {};
```

Comment:
Why `therapeuticLine` was renamed to `arvPlan`. Note that those are two different thing in use here.